.gitignore

```
@@ -30,4 +30,4 @@ qml_debug.log
 datasets/*
 .benchmarks/*
 *.h5
-*.hdf5
+*.hdf5
```

I think files need to end with a newline for pylint to be happy. 😄 
```suggestion
*.hdf5

```